vm/src/builtins/bytearray.rs

```diff
@@ -91,6 +91,13 @@ impl PyPayload for PyByteArray {
 
 /// Fill bytearray class methods dictionary.
 pub(crate) fn init(context: &Context) {
+    context
+        .types
+        .bytearray_type
+        .slots
+        .hash
+        .store(Some(unhashable_wrapper));
```

Comment:
I have a suggestion about the location of this initialization.

We put these information to `PyClassDef` for builtin types.
adding `HASHABLE` const with default value true will be good.

Then we use the information during making class. that's `PyClassImpl::make_slots`.

Reply:
Thanks, I'll try.

Reply:
maybe default value is false, not true. hashable is common in builtin type world, but not outside.